apps/src/templates/sectionsRefresh/VersionUnitDropdowns.jsx

```diff
@@ -8,7 +8,8 @@ import _ from 'lodash';
 export default function VersionUnitDropdowns({
   courseOffering,
   updateCourse,
-  sectionCourse
+  sectionCourse,
+  isNewSection
```

Comment:
Curious about why we needed to pass 'isNewSection' through!

Reply:
Yep - this doesn't need to be here!